Sources/Shared/Operators/Operators.swift

```diff
@@ -32,3 +33,7 @@ public func <+ <T>(inout left: [T], right: T?) {
   guard let right = right else { return }
   left.append(right)
 }
+
+public prefix func <? <T: DefaultType>(value: T?) -> T {
+  return value ?? T.defaultValue
+}
```

Comment:
I tried to use infix operator with default value https://github.com/onmyway133/issues/issues/136
Declaration of that function operator is OK, but when using that function, the compiler is not happy